dt/host/InspectorFrontendHost.js

```diff
@@ -187,7 +229,19 @@ WebInspector.InspectorFrontendHostStub.prototype = {
      */
     append: function(url, content)
```

Comment:
DT use "save" to check if file is writtable and write content in chunks using "append"